bugguide/bugguide
# #40 style browse tab on guide pages

### Description

Fix classification titles in a group of images under browse tab

### Testing steps

*   [ ] Go to /node/356965/bgpage
*   [ ] Notice the title should follow the pattern from #25

![](https://user-images.githubusercontent.com/1582129/101839728-e1dffb00-3b10-11eb-827c-8fd8415f35e6.png)
